client/src/theme/overrides/Link.js

```diff
@@ -9,22 +9,28 @@
 export default function Link(theme) {
   return {
     MuiLink: {
+      defaultProps: {
+        default: "primary"
```

Comment:
@93Belen just noticed this. It should be `variant: "primary"` 😅

Reply:
https://github.com/hackforla/food-oasis/pull/1720